theseus/core/tests/test_objective.py

```diff
@@ -286,8 +286,10 @@ def _check_error_for_data(v1_data_, v2_data_, error_, error_norm_2_):
         v2_data = torch.ones(batch_size, dof) * f2 * 0.1
 
         input_data = {"v1": v1_data, "v2": v2_data}
-        error = objective.error(input_data=input_data, also_update=True)
-        error_norm_2 = objective.error_squared_norm(input_data=input_data, also_update=True)
+        error = objective.error(input_data=input_data, also_update=False)
+        error_norm_2 = objective.error_squared_norm(
+            input_data=input_data, also_update=False
+        )
```

Comment:
We might need a check after this that makes sure the older variables are still there and didn't get updated?

Reply:
Oh yes, good catch, let's add this.

Reply:
Oh yes, good catch, let's add this.